ros_gz_bridge/launch/ros_gz_bridge.launch.py

```diff
@@ -46,6 +47,12 @@ def generate_launch_description():
         description='Name of container that nodes will load in if use composition',
     )
 
+    declare_start_container_cmd = DeclareLaunchArgument(
+        'start_container',
```

Comment:
I would suggest changing this to `create_own_container` to make it a little more clear that this is not necessary for the bridge to work as a composable node. I would also add more to the description so users don't think they have to set this to True in order to get the benefits of composition.

Reply:
I renamed the new parameter as suggested and added to the launch action. I also changed the default value of `use_composition` to `False`. [f0f63b3](https://github.com/gazebosim/ros_gz/pull/616/commits/f0f63b3d9182506687cd88a14bf93ea726afaec6)